### DISCUSSION TITLE
how do i delete a user from the supabase Authentication-user table with sveltekit?

### DISCUSSION THREAD

**Discussion by FAalInnovation** [supabase/supabase#13973] [Questions] [answered]
how do i delete a user from the supabase user table with sveltekit?

**Comment by GaryAustin1** [comment, marked as answer]
Not at all clear what you are trying to do.
Sveltekit is a very broad framework.
There is an admin auth delete user call that can be run from server side only with service_role.
Auth schema cannot be accessed from client code.  You have to use rpc calls to a postgres function and the function has to be security definer type.
